ddl/column.go

```diff
@@ -570,9 +570,16 @@ func generateOriginDefaultValue(col *model.ColumnInfo) (interface{}, error) {
 		}
 	}
 
-	if odValue == strings.ToUpper(ast.CurrentTimestamp) &&
-		(col.Tp == mysql.TypeTimestamp || col.Tp == mysql.TypeDatetime) {
-		odValue = time.Now().Format(types.TimeFormat)
+	if odValue == strings.ToUpper(ast.CurrentTimestamp) {
+		if col.Tp == mysql.TypeTimestamp {
+			odValue = time.Now().UTC().Format(types.TimeFormat)
+			// Version = 1: For OriginDefaultValue and DefaultValue of timestamp column will stores the default time in UTC time zone.
+			//              This will fix bug in version 0.
+			// TODO: remove this version field after there is no old version 0.
+			col.Version = model.ColumnInfoVersion1
```

Comment:
It is also used in `onDropColumn`, this may have compatibility issues with previous(ColumnInfoVersion0) and future(ColumnInfoVersion1) versions?

Reply:
Right,  but we can only cancel drop column DDL job when the job is not running, So I think it doesn't matter because the client will never see the dropped column value.

Reply:
Yes, but I think if we support cancel "drop column" operation later, we may forget this. Maybe we need to add comments for it.

Reply:
Already add a comment for this.